### ISSUE TITLE
Battletest the ZGW API-connection between Rxmission and Open Formulieren

### ISSUE THREAD

**Issue by joeribekker** [open-formulieren/open-forms#3649]
### Thema / Theme

Other

### Omschrijving / Description

There are few unexpected issues when using Rxmission as ZGW API registration backend.

Where we typically read the OAS to adjust our calls to the backend, this concept came unexpected to Visma Roxit. The OAS was not readily available for retrieval. After manually adding the OAS, some other issues came to light, like passing an API-version header attribute `bestandsomvang` that didn't match the upload size (for documents).

These issues needs to be tracked, analysed and fixed. Since the ZGW API in Rxmission is also subject to changes, we might ask them to fix things or workaround issues for now.

### Added value / Toegevoegde waarde

Harden the ZGW API registration backend to be used in more infra setups.

### Aanvullende opmerkingen / Additional context

No claims are made who is correct or not, we just want things to work :)

### Tasks

* [ ] Add version selection option to `Service`. This value can be empty. If filled, it should have an effect on the `API-version` header that is sent.
* [ ] Figure out why the error doesn't show up in request-logging

By setting API-version to 1.0 for Documenten API, this should work on Rxmission.

**Comment by joeribekker** [comment]
Discussion with Roxit. Sending the "bestandsomvang" attribute is required for their Documenten API in 1.1 and up. It is not like that in the standard... so meh.

**Comment by sergei-maertens** [comment]
Standard allows for `null`, so can we just send that? In older versions this field is a calculated value, so Open Zaak will ignore it if its sent.

**Comment by joeribekker** [comment]
@SilviaAmAm I added tasks in the description. The idea is to add Version selection and that we sent the API-version header. This solves the immediate issue AND is a nice feature overall I think, that we can extend later.

If we sent the 1.0 version, everything works. If we sent the 1.1 version, bestandsgrootte is required (and it shouldn't) but anyway, we can then set it to 1.0 to make it work.

I already informed them and they created a bug report internally.
